core/src/main/scala/org/apache/spark/util/random/SamplingUtils.scala

```diff
@@ -108,14 +111,22 @@ private[spark] object SamplingUtils {
 private[spark] object PoissonBounds {
 
   /**
-   * Returns a lambda such that Pr[X > s] is very small, where X ~ Pois(lambda).
+   * Returns a lambda such that
+   * {{{
+   * Pr[X > s]
+   * }}}
+   * is very small, where X ~ Pois(lambda).
```

Comment:
- Java
  <img width="391" alt="2016-11-27 3 24 47" src="https://cloud.githubusercontent.com/assets/6477701/20642393/2575a0ae-b451-11e6-88da-caf18f2b587b.png">


Scaladoc not found